backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -110,6 +111,12 @@ class Meta:
     thesis_info = fields.Nested(ThesisInfoSchemaV1, dump_only=True)
     dataset_links = fields.Method("get_datasets")
 
+    def is_collection_hidden(self, data):
+        collections = data.get("_collections", [])
+        if "Literature" in collections and len(collections) > 1:
```

Comment:
I think that this check is not correct. We have following options:
* length of collections is one and the only collection is `Literature` --> not hidden
* length of collections is one but `Literature` is not there --> hidden
* length is more than 1 --> then there is for sure at least one hidden.
For now you're handling correctly only case 3.